_notices/rgn0013.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: notice
+parent: RAPIDS General Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rgn
+# Update meta-data for notice
+notice_id: 13 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "RAPIDS switching to CalVer in v0.20"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Release Version Change
+notice_rapids_version: "v0.20"
+notice_created: 2021-05-05
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2021-05-05
+---
+
+## Overview
+
+All RAPIDS projects are switching their versioning scheme to [Calendar Versioning](https://calver.org).
+
+## Impact
+
+Instead of a `0.20.0` release in June 2021, we will release `21.06.00`
+
+New version format for release: `YY.MM.PP`, for example `21.06.00`
+
+New version format for nightlies: `YY.MM.PPaYYMMDD`, for example `21.06.00a210505`
+
+Where `PP` is the two digit patch number not day.
```

Comment:
Add a line:

"This versioning will apply to both the overall RAPIDS release and each core library, so we will release cuDF 21.06.00, cuML 21.06.00, etc. as part of the overall RAPIDS 21.06.00 release."

(Just since this question came up in internal discussion too)

Reply:
Done